### ISSUE TITLE
not working on Windows Server 2019

### ISSUE THREAD

**Issue by outstanding1301** [F1bonacc1/process-compose#90]
## Defect

Make sure that these boxes are checked before submitting your issue -- thank you!

- [x] Included the relevant configuration snippet
- [x] Included the relevant process-compose log (log location: `process-compose info`)
- [x] Included a [Minimal, Complete, and Verifiable example] (https://stackoverflow.com/help/mcve)

#### Expected result:
``` 
Process Compose
Version:        v0.60.0
Commit:         79d0cbd
Date (UTC):     2023-07-21T21:40:41Z
License:        Apache-2.0

Written by Eugene Berger
```

#### Actual result:
```
panic: open C:\Users\ADMINI~1\AppData\Local\Temp\2\process-compose-WIN-2B62Q54U8RC\Administrator.log: The system cannot find the path specified.

goroutine 1 [running]:
github.com/f1bonacc1/process-compose/src/cmd.Execute()
        /home/eugene/projects/go/process-compose/src/cmd/root.go:54 +0x111
main.main()
        /home/eugene/projects/go/process-compose/src/main.go:9 +0x17
```

---

Full command

```bat
C:\Users\Administrator\Downloads>process-compose.exe version
panic: open C:\Users\ADMINI~1\AppData\Local\Temp\2\process-compose-WIN-2B62Q54U8RC\Administrator.log: The system cannot find the path specified.

goroutine 1 [running]:
github.com/f1bonacc1/process-compose/src/cmd.Execute()
        /home/eugene/projects/go/process-compose/src/cmd/root.go:54 +0x111
main.main()
        /home/eugene/projects/go/process-compose/src/main.go:9 +0x17
```

When I manually create the `C:\Users\ADMINI~1\AppData\Local\Temp\2\process-compose-WIN-2B62Q54U8RC` directory, it works.

I think there's a problem creating the directory!

> BTW, thank you for this beautiful app. It is exactly what I looking for! 😆

**Comment by F1bonacc1** [comment]
Hi @outstanding1301,

Thank you for letting me know!
This should be an easy fix and will be delivered in the next version.

Glad you like the app! I'm glad to see that all the efforts to keep the cross-platform compatibility weren't wasted :)

**Comment by F1bonacc1** [comment]
Fixed in v0.65.1
